examples/demo11.py

```diff
@@ -209,10 +209,12 @@ def main(argv):
     theta = 0.17 * galsim.degrees
     # ( dudx  dudy ) = ( cos(theta)  -sin(theta) ) * pixel_scale
     # ( dvdx  dvdy )   ( sin(theta)   cos(theta) )
-    dudx = math.cos(theta.rad()) * pixel_scale
-    dudy = -math.sin(theta.rad()) * pixel_scale
-    dvdx = math.sin(theta.rad()) * pixel_scale
-    dvdy = math.cos(theta.rad()) * pixel_scale
+    # Aside: You can call numpy trig functions on Angle objects directly, rather than getting
```

Comment:
That's cool.  Question: later on I see exactly one place (galsim/config/value.py) where you do `theta.cos()`.  Is there any effective difference (speed or whatever) compared to doing `numpy.cos(theta)`?  Also, may be worth mentioning both options here?